From 21bf95659e95cac91fb08931e6689b1eaddec5c6 Mon Sep 17 00:00:00 2001
From: William Woodall <william@osrfoundation.org>
Date: Wed, 15 Oct 2014 11:09:12 -0700
Subject: [PATCH] specify the -DNDEBUG option for performance

---
 debian/rules.em | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/rules.em b/debian/rules.em
index 06777bd..a06c8a0 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -15,6 +15,8 @@ export DH_OPTIONS=-v --buildsystem=cmake
 #  https://code.ros.org/trac/ros/ticket/3842
 export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
+# Manually specify the -DNDEBUG option for performance reasons
+export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
 
 %:
 	dh  $@@
-- 
1.9.1

