From 77128651015d4de87db911bb2dff915467a10075 Mon Sep 17 00:00:00 2001
From: William Woodall <william@osrfoundation.org>
Date: Tue, 7 Oct 2014 15:04:49 -0700
Subject: [PATCH 3/4] move comment (again)

---
 debian/rules.em | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 633c7a7..ab55770 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -24,7 +24,6 @@ override_dh_auto_configure:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	# Use release mode explicitly for performance
 	dh_auto_configure -- \
 	        -DCMAKE_BUILD_TYPE="Release" \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
-- 
1.9.3 (Apple Git-50)

