From 66f3d90b6e2cf536192ea07eb8015edda3f3d0d7 Mon Sep 17 00:00:00 2001
From: William Woodall <william@osrfoundation.org>
Date: Wed, 15 Oct 2014 11:08:05 -0700
Subject: [PATCH 4/4] explicitly only set -DNDEBUG for releasing rviz

---
 debian/rules.em | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index ab55770..a06c8a0 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -15,6 +15,8 @@ export DH_OPTIONS=-v --buildsystem=cmake
 #  https://code.ros.org/trac/ros/ticket/3842
 export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
+# Manually specify the -DNDEBUG option for performance reasons
+export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
 
 %:
 	dh  $@@
@@ -25,7 +27,6 @@ override_dh_auto_configure:
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_configure -- \
-	        -DCMAKE_BUILD_TYPE="Release" \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
-- 
2.2.1

